app/routes.py

```diff
@@ -1,2 +1,110 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, make_response, request
 
+# path = "https://api.le-systeme-solaire.net/rest.php/bodies"
+
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+
+@planets_bp.route("", methods=["GET", "POST"])
+def handle_planets():
+    if request.method == "GET":
+        name = request.args.get("name")
+        if name:
+            planets = Planet.query.filter_by(name=name) 
+        else:
+            planets = Planet.query.all()
+        planets_response = []
+        for planet in planets:
+            planets_response.append(
+                {
+                    "id": planet.id,
+                    "name": planet.name,
+                    "description": planet.description,
+                    "color": planet.color
+                }
+            )
+        return jsonify(planets_response), 200
+
+    elif request.method == "POST":
+        request_body = request.get_json()
+        new_planet = Planet(name=request_body["name"],
+                            description=request_body["description"],
+                            color=request_body["color"]
+                            )
```

Comment:
We should check if `request_body` contains falsy values so that a planet isn't created with empty data!